lib/cli.js

```diff
@@ -21,6 +21,10 @@ Options:
     generate source maps (where supported)
   --compact
     reduce output size (where supported)
+  --serve [HOST:]PORT
+    serve the files via HTTP
+  --liveserve [HOST:]PORT
+    serve the files via HTTP with live reload
```

Comment:
Which files, exactly? Doesn't seem obvious for users, particularly if there's no explicit `webRoot`.

Reply:
Yes, you are right. But not sure how to explain that so briefly... Maybe "the generated files"?

Reply:
I went with "serve generated files"; good enough, I suppose. 👌